demo/src/screens/HomeScreen/HomeCarousel/HomeCarousel.js

```diff
@@ -46,10 +166,26 @@ export function HomeCarousel() {
           alt="Third slide"
         />
         <Carousel.Caption>
-          <h3 className="font-italic">Toe Beans</h3>
-          <p>{`Seen like never before. `}
-            <Link to="/product/3h488">Buy Now</Link>
-          </p>
+          <div className="header-with-modal carousel-header">
+            <h3 className="font-italic">
+              Toe Beans
+            </h3>
+            {carouselCodeModal}
+          </div>
+          <div className="header-with-modal carousel-text">
+            <p>
+              {`Seen like never before. `}
```

Comment:
Why "Toe Beans" is used without {} but this line requires wrapping?

I'd recommend being consistent with across the codebase and probably establish general rules where one or the other style should be used and agree with all team members to follow it.

Reply:
I usually use template strings in JSX to preserve whitespace as a lot of code editors will auto trim these on save. This one has a trailing space that is necessary whereas Toe Beans does not. I am fine using them all over to keep things consistent and future-proof though so I will make the appropriate edits.